init.lua

```diff
@@ -5,14 +5,39 @@ local MODPATH = minetest.get_modpath("lavastuff")
 local COOLDOWN = dofile(MODPATH.."/cooldowns.lua")
 local S
 
-if minetest.get_translator ~= nil then
-	S = minetest.get_translator(minetest.get_current_modname())
-else
-	S = function(str)
-		return(str)
-	end
-end
-
+if minetest.get_translator ~= nil then
+	S = minetest.get_translator("mobs")
+else
+	if minetest.get_modpath("intllib") then
```

Comment:
You should add an optional dependency to `mod.conf` for this too

Put it on the first line, I've reserved that for non-game mods like 3d_armor

Reply:
the dependency in mod.conf is ilogical.. . cos since 5.0.X (mod.conf is only detected in 5.X releases) the intlibs mod is not used.. that's why the detection of minetest.get_translator there.. only for 4.X (depends.txt) is used and only in that file is present

explained, in commit [9cac4ad](https://github.com/minetest-mods/lavastuff/pull/16/commits/9cac4ad93195a50ab2a1a0c4292d0ac329bef730)